docs/index.md

```diff
@@ -33,38 +33,38 @@ Within this Handbook you will find topics on history, lore, guidelines for behav
+With the creation of the Iron Symphony & the Anvil Coalition, the Second Life Military Community faced a new threat; Alliances of power-hungry leaders who would not leave a game as just a game, those who wished to dictate how the community would exist for the unforeseeable future. These Alliances were spearheaded by the Ordo Imperialis & Vanguard, rival powers on the grid at the time, would attack dissenting groups internally and externally, without regards to what is and is not morally acceptable. The Second Life Military Community faced an era of political strife and drama, outweighing the fun of it. Thanks to these groups, their attacks led to a massive drop in SLMC activity.
 
-During this Era, one banner was raised against these ideals. The banner of Decretum. The proud members of Chaos Decretum strode forth and fought against the political alliances and started a campaign of revitalizing destruction of these groups.
+During this Era, one banner was raised against these ideals; the banner of Decretum. The proud members of Chaos Decretum strode forth and fought against the political alliances and started a campaign to fuel the destruction of these groups.
 
-Spiteful as it was, it was necessary at the time. However a campaign of destruction often turns inwards once there are no remaining targets. In the midst of the campaign Chaos Decretum fell, but not after inflicting considerable damage to these factions.
+Spiteful as it was, it was necessary at the time. However, a campaign of destruction often turns inwards once there are no remaining targets. In the midst of the campaign Chaos Decretum fell, but not after inflicting considerable damage to these factions.
 
-Years after the death of Chaos Decretum, Chaos Eternus was born, forging a hellfire once more. Jaded from past failures the Dominus' goal was to destroy what he could and to conquer - not through tactics but through firepower. Such is how the notoriety of Chaos was earned. Through yet another campaign of hatred was the grid dragged, one without an enemy clearly in sight.
+Years after the death of Chaos Decretum, Chaos Eternus was born, forging a hellfire once more. Jaded from past failures, the Dominus' goal was to destroy what he could and to conquer - not through tactics, but through sheer firepower. Such is how the notoriety of Chaos was earned. Through yet another campaign of hatred was the grid dragged through, one without an enemy clearly in sight.
```

Comment:
I suggesting making this something like 

`The Grid was dragged through another campaign of hatred, `

Or something. Double **through** looks weird.